code/iaas/auth-logic/src/main/java/io/cattle/platform/iaas/api/auth/integration/ldap/OpenLDAP/OpenLDAPConfig.java

```diff
@@ -17,6 +17,7 @@
     private final Integer userDisabledBitMask;
     private final String loginDomain;
     private final String domain;
+    private final String groupDomain;
```

Comment:
Can you rename this to "groupSearchDomain" to make it clear that we will apply this in group search function